tstest/natlab/natlab_test.go

```diff
@@ -235,28 +233,84 @@ func TestFirewall(t *testing.T) {
 	trust := m.Attach("trust", lan)
 	untrust := m.Attach("untrust", wan)
 
-	f := &Firewall{
-		TrustedInterface: trust,
-		SessionTimeout:   30 * time.Second,
-		TimeNow:          clock.Now,
-	}
-
 	client := ipp("192.168.0.2:1234")
 	serverA := ipp("2.2.2.2:5678")
-	serverB := ipp("7.7.7.7:9012")
-	tests := []struct {
-		iface    *Interface
-		src, dst netaddr.IPPort
-		want     PacketVerdict
-	}{
-		{trust, client, serverA, Continue},
-		{untrust, serverA, client, Continue},
-		{untrust, serverA, client, Continue},
-		{untrust, serverB, client, Drop},
-		{trust, client, serverB, Continue},
-		{untrust, serverB, client, Continue},
-	}
+	serverB1 := ipp("7.7.7.7:9012")
+	serverB2 := ipp("7.7.7.7:3456")
+
+	t.Run("ip and port dependent", func(t *testing.T) {
```

Comment:
I prefer no spaces in subtest names. Makes it harder to run from shell with `-run=regexp`